notebooks/examples/configure_check_conditions.ipynb

```diff
@@ -0,0 +1,533 @@
+{
```

Comment:
<p>dot at the end of both sentences</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>